conf/tls/tls_config.go

```diff
@@ -0,0 +1,125 @@
+package tls
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+
+	"github.com/squareup/pranadb/errors"
+
+	"github.com/alecthomas/kong"
+)
+
+// TLSConfig can be used to build a TLS configuration.
+type TLSConfig struct {
+	// nolint: golint
+	CACert     NamedFileContent `help:"PEM-encoded CA certificates to use for ${tls_target}." placeholder:"FILE"`
+	Key        NamedFileContent `help:"Private PEM-encoded key to use for ${tls_target}. Only use if unsetting --key-pair." placeholder:"FILE"`
```

Comment:
It's unclear whether these are actual keys or paths to key files on the filesystem. It would be very helpful if there was a usage example. Could you create an example config file in the cfg directory to show usage?

Reply:
Done. Added it to `local-deployment/conf/pranadb-conf`. This is the deployment using Docker compose.

The API TLS can be configured inside the PranaDB config HCL or overridden via CLI arguments using `--api-tls-ca-cert`, `--api-tls-key`. The PranaDB conf struct uses Kong's struct `embed` tag with a prefix `api-tls-`.